automation/project/Dependencies.scala

```diff
@@ -58,14 +59,15 @@ object Dependencies {
       excludeGoogleJsr305,
       excludeJacksonCore),
 
+    guava,
```

Comment:
Just curious what we're using guava for.

Reply:
needed for google2 update

Reply:
it's not needed explicitly anywhere. But it's used by several google related libraries. Forcing the version here to solve an issue somehow a lower version is brought in which lead to compile error.